btk/draw_blends.py

```diff
@@ -188,7 +216,15 @@ def run_mini_batch(Args, blend_list, obs_cond):
             )
             blend_image_multi[:, :, j] = single_band_output[0]
             iso_image_multi[:, :, :, j] = single_band_output[1]
-        mini_batch_outputs.append([blend_image_multi, iso_image_multi, blend_list[i]])
+        wcs = make_wcs(
```

Comment:
did you mean to have it inside the for loop?

Reply:
Good point. Actually right now the WCS information is strictly the same for each band so it does not matter from which one it is taken. Depending on how we implement multi-resolution this may be relevant later ; as of now changing the "j" index by 0 would be more clear maybe ?

Reply:
Yeah maybe it's best to have it outside and index by 0 to make this explicit, and maybe have a comment just saying that it's currently the same across bands.